sqlite3/lib/src/api/exception.dart

```diff
@@ -21,14 +21,24 @@ class SqliteException implements Exception {
   /// code, providing some idea of the cause of the failure.
   int get resultCode => extendedResultCode & 0xFF;
 
-  SqliteException(this.extendedResultCode, this.message, [this.explanation]);
+  /// Last SQL command executed
+  String sqlCommand;
```

Comment:
This should be final, I also think we should use "statement" instead of "command":

```suggestion
  /// The SQL statement triggering this exception.
  final String? sqlStatement;
```

Reply:
Done